chaos/ids.go

```diff
@@ -0,0 +1,30 @@
+package chaos
+
+import (
+	"github.com/filecoin-project/go-address"
+	"github.com/filecoin-project/specs-actors/actors/builtin"
+	"github.com/ipfs/go-cid"
+	"github.com/multiformats/go-multihash"
+)
+
+// ChaosActorCodeCID is the CID by which this kind of actor will be identified.
+var ChaosActorCodeCID = func() cid.Cid {
+	builder := cid.V1Builder{Codec: cid.Raw, MhType: multihash.IDENTITY}
+	c, err := builder.Sum([]byte("fil/1/chaos"))
+	if err != nil {
+		panic(err)
+	}
+	return c
+}()
+
+// Address is the singleton address of this actor. It is value 97
+// (builtin.FirstNonSingletonActorId - 3), as 99 is reserved for the burnt funds
+// singleton, and 98 is the puppet actor.
```

Comment:
Is the puppet actor needed after we have the chaos actor? Can we just use 98?

Is it possible for the chaos actor to just _be_ a puppet actor rather than use another ID?

Reply:
We could merge both, or even embed the Puppet actor inside the Chaos actor. But whatever we choose, it needs to live in our land and not in specs-actors, as it's going to evolve alongside the test vectors themselves.